README.md

```diff
@@ -562,9 +585,9 @@ common for startups.
 contracts that allow you to buy shares in the company you work for.
 When you buy shares, you are **exercising** the options.
 
-- 🔸 Options are not the same as stock;
-  they are only the right to buy stock upon and subject to the conditions specified in the
-  option agreement.
+☝️ Options are not the same as stock;
```

Comment:
Even more clarity: 'Stock options are just that: the _option to buy stock_ at a certain price (the strike price) on a certain schedule (the vesting schedule). If you have stock options, you are not a shareholder until you _exercise_ (or purchase) your stock. Now we'll dive into the details of all of that.'